bindata/scripts/load-kmod.sh

```diff
@@ -1,11 +1,12 @@
 #!/bin/sh
 # chroot /host/ modprobe $1
 kmod_name=$(tr "-" "_" <<< $1)
+kmod_args="${@:2}"
```

Comment:
will this also work if we only pass 1 arg?

Reply:
```bash
~ cat test.sh 
#!/bin/sh
# chroot /host/ modprobe $1
kmod_name=$(tr "-" "_" <<< $1)
kmod_args="${@:2}"
echo "Loading module $kmod_name with args $kmod_args"
```

```bash
~ ./test.sh foo bar
Loading module foo with args bar
```

``` bash
~ ./test.sh foo    
Loading module foo with args 
```

```bash
./test.sh foo arg1 arg2 arg3
Loading module foo with args arg1 arg2 arg3
```